sormas-api/src/main/java/de/symeda/sormas/api/task/TaskCriteria.java

```diff
@@ -26,14 +26,20 @@
 import de.symeda.sormas.api.contact.ContactReferenceDto;
 import de.symeda.sormas.api.event.EventReferenceDto;
 import de.symeda.sormas.api.person.PersonReferenceDto;
+import de.symeda.sormas.api.region.DistrictReferenceDto;
+import de.symeda.sormas.api.region.RegionReferenceDto;
 import de.symeda.sormas.api.user.UserReferenceDto;
 import de.symeda.sormas.api.utils.IgnoreForUrl;
 
 public class TaskCriteria extends BaseCriteria implements Serializable {
 
 	private static final long serialVersionUID = -9174165215694877625L;
 
+	public static final String CONTEXT_ENTITY_FREE_TEXT = "contextEntityFreeText";
+
 	private TaskStatus taskStatus;
+	private TaskContext taskContext;
+	private String contextEntityFreeText;
```

Comment:
The name is a bit misleading for me because it's not about the context entity; I'd rather just call it "freeText"